RAML/ez.raml

```diff
@@ -0,0 +1,1277 @@
+                                                  "email": "john.doe@example.net"
+                                                },
+                                                {
+                                                  "name": "Bruce Willis",
+                                                  "email": "bruce.willis@example.net"
+                                                }
+                                              ]
+                                            }
+                                          ]
+                                        }
+                                      }
+                                    },
+                                    "Section": {
+                                      "_href": "/content/sections/4",
+                                      "_media-type": "application/vnd.ez.api.Section+json"
+                                    },
+                                    "MainLocation": {
+                                      "_href": "/content/locations/1/4/65",
+                                      "_media-type": "application/vnd.ez.api.Location+json"
+                                    },
+                                    "Locations": {
+                                      "_href": "/content/objects/23/locations",
+                                      "_media-type": "application/vnd.ez.api.LocationList+json"
+                                    },
+                                    "Owner": {
+                                      "_href": "/user/users/14",
+                                      "_media-type": "application/vnd.ez.api.User+json"
+                                    },
+                                    "lastModificationDate": "2012-02-12T12:30:00",
+                                    "mainLanguageCode": "eng-US",
+                                    "currentVersionNo": "1",
+                                    "alwaysAvailable": true
+                                  }
+                                }
+                400:
+                    description: Error - if the Input does not match the input schema definition or the validation on a field fails
+                401:
+                    description: Error - if the user is not authorized to create this object in this location
+                404:
+                    description: Error - if a parent location is specified in the request body and it does not exist
+        get:
+            description: Loads the content for a given remote ID
+            queryParameters:
+                remoteId:
+                    description: The remote ID of the content. If present the content with the given remote ID is returned
+            responses:
+                307:
+                    description: Temporary redirect
+                404:
+                    description: Error - if the content with the given remote ID does not exist
+    /objects/{id}:
```

Comment:
Nitpick: this might be a good chance to improve readability (either now or as a follow-up). Maybe `{contentId}` instead, so we exactly know what that is.